docs/switch.md

```diff
@@ -95,7 +95,17 @@ Invoked when the user tries to change the value of the switch. Receives the new
 
 ### `thumbColor`
 
-Color of the foreground switch grip. If this is set on iOS, the switch grip will lose its drop shadow.
+Color of the default foreground switch grip. If this is set on iOS, the switch grip will lose its drop shadow.
+
+| Type               |
+| ------------------ |
+| [color](colors.md) |
+
+---
+
+### `activeThumbColor`
```

Comment:
Could you link to where this is defined in React Native?

Reply:
I can't find the proper documentation for this, But when I was having an issue to manage the activeThumbColor, I searched for it and found that there is a prop accepted by the Switch component and it is not in the documentations and not in the types of the Switch component.

I have raised PR for the type changes in react-native repo also.

Reply:
Could you link your other PR then?

Reply:
How can I link my other PR?
Could you please help me with that?

Here is the PR link: https://github.com/facebook/react-native/pull/38636

Reply:
That's sufficient so we can merge this one once your other PR is merged

Reply:
Hi @cortinico The above PR merged! 😊
Kindly check